packages/services/video-tools/video_utils/ai_commands/video_commands.py

```
@@ -0,0 +1,661 @@
+                result = _analyze_with_gemini(video_file, model)
+
+            output_name = video_file.stem + "_detailed_timeline"
+            output_file = paths.output_dir / output_name
+
+            if _save_timeline_result(result, output_file):
+                successful += 1
+                results.append(result)
+            else:
+                failed += 1
+
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)